### DISCUSSION TITLE
GIZ GHG inventory data

### DISCUSSION THREAD

**Discussion by juka19** [transport-data/community#26] [Sources]
Hi all,

so GIZ uses a couple of spreadsheet-based toolkits to create transport GHG inventories ([TrIGGER](https://changing-transport.org/tool/trigger/), developed by ifeu, [Mobilise Your City Emissions Calculator](https://changing-transport.org/tool/mobiliseyourcity-emissions-calculator/), and some more). While the spreadsheets are oftentimes adapted quite heavily, I do think it is an opportunity to adapt the [gfei-2023 script](https://github.com/transport-data/gfei-2023/blob/main/create.py) because these toolkits are used fairly widely by our projects, and I suppose possibly elsewhere.

It is pretty straight forward to do this for the road transport ([see here](https://github.com/transport-data/TRiGGER-Kenya-2015/tree/main))- however I am a bit clueless how to do so with regards to the other transport modes (maritime, aviation, railways). I was wondering whether there is some type of standard I can adhere to @khaeru?

**Comment by khaeru** [comment]
> I was wondering whether there is some type of standard I can adhere to

transport-data/tools#9 will add some initial, general standards. But for some other concepts like `MODE`, there is no single accepted common code list. I don't think TDCI should necessarily be the place to maintain those, unless the community wants to pursue that. Where there are well-known, existing structures, TDC should reference/use those (see e.g. transport-data/tools#7) and encourage others to do the same.

iTEM has [these structures](https://github.com/transportenergy/database/blob/main/item/structure/base.py). These are neither final nor well-documented, but I think by using the same concept IDs and codes is probably a code start and will be easy to later update. For instance, [here](https://github.com/transportenergy/database/blob/68f9fe0109c92b3fecb0e8a5f1372527d77b9891/item/structure/base.py#L311-L331) is the code list used for the `MODE` concept.
